PWGCF/FemtoUniverse/Core/FemtoUniverseDetaDphiStar.h

```diff
@@ -225,8 +225,6 @@ class FemtoUniverseDetaDphiStar
       for (int i = 0; i < 2; i++) {
         auto indexOfDaughterpart1 = (ChosenEventType == femtoUniverseContainer::EventType::mixed ? part1.globalIndex() : part1.index()) - 2 + i;
         auto indexOfDaughterpart2 = (ChosenEventType == femtoUniverseContainer::EventType::mixed ? part2.globalIndex() : part2.index()) - 2 + i;
-        // auto indexOfDaughterpart1 = part1.globalIndex() - 2 + i;
```

Comment:
Remove unused code

Reply:
This is just a comment for later PR, I can approve it without this change. Crucial thing for me is green light from @alicja-pp @basiach and @mm1707.

Reply:
I see that @alicja-pp and @mm1707 agreed (@basiach is not available) so I am approving the changes.